s3tests_boto3/functional/test_s3select.py

```diff
@@ -15,6 +15,10 @@
 import logging
 logging.basicConfig(level=logging.INFO)
 
+#TODO the following resolve an issue with python version (should be conditional)
```

Comment:
@galsalomon66 can you resolve or remove this TODO before we merge this?

Reply:
:+1: